apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -744,6 +750,33 @@ export default Vue.extend({
       pendingUpdates.push(payload)
       this.$set(this.pendingChanges, 'updates', pendingUpdates)
     },
+    cellCopyRowAsInsert(e, cell) {
+      const row = cell.getRow()
+      const data = { ...row.getData() }
+      const payload = {
+        table: this.table.name,
+        schema: this.table.schema,
+        data
+      }
+      const insertQuery = this.createInsertStatement(payload);
+      this.$native.clipboard.writeText(insertQuery);
+    },
+    createInsertStatement(payload) {
```

Comment:
Nice! Love this functionality. However I'm trying to avoid building SQL in the front-end as it doesn't always work the same across all databases.

Would you be open to adding a new method to the `Connection` interfaces to support this? Something like `buildInsertStatements(table, values, schema?)`, it would need implementing on postres, sql-server, mysql, and sqlite, although probably a generic version can be created and shared between them.